app/common/auth/authentication-events.run.js

```diff
@@ -20,10 +20,14 @@ function AuthenticationEvents($rootScope, $location, Authentication, Session, _,
     function doLogin(redirect, noReload) {
         loadSessionData();
         $rootScope.loggedin = true;
-        if (redirect) {
-            $location.url(redirect);
-        }
-        noReload || $route.reload();
+        TermsOfServiceEndpoint.get()
```

Comment:
Could we hide the endpoint entirely inside the service? ie. so you'd just call
`TermsOfService.tosCheck().then(....`